src/sagas/admin.js

```diff
@@ -103,6 +110,7 @@ export function* deployStore({ payload }) {
     );
 
     if (accountBalance >= transactionCostInEthers) {
+      toast("Please confirm the transaction on your wallet/ledger.");
```

Comment:
I didnt try this code, but I'm 99% sure this wont wait the toast to be closed => the code below will always run

Reply:
Think that is his intention